blenderAddons/vispCAOExport/setup.py

```diff
@@ -0,0 +1,9 @@
+import bpy
+import sys
+import os
+
+print("BLENDER VERSION : " + bpy.app.version_string)
+ADDON_PATH = (bpy.utils.user_resource('SCRIPTS', "addons"))
+
+os.system("cp -r ../vispCAOExport " + ADDON_PATH)
+print("Moved addon to " + ADDON_PATH)
```

Comment:
Oops, this was a bad script. I wrote this for temporary purpose.

The project can actually be added from blender directly from the user preference menu. Since it has to be visited anyways to enable the plugin, I think there is no need for a setup script. The folder has to be zipped though to install from blender.